python/ray/ml/train/integrations/torch/torch_trainer.py

```diff
@@ -188,3 +191,32 @@ def __init__(
             preprocessor=preprocessor,
             resume_from_checkpoint=resume_from_checkpoint,
         )
+
+    @staticmethod
+    def load_checkpoint(
+        checkpoint: Checkpoint, model: Optional[torch.nn.Module] = None
+    ) -> Tuple[torch.nn.Module, Optional[Preprocessor]]:
+        """Load a Checkpoint from ``TorchTrainer``.
+
+        Return the model with set weights and AIR preprocessor contained within.
+
+        Args:
+            checkpoint: The checkpoint to load the model and
+                preprocessor from. It is expected to be from the result of a
+                ``TorchTrainer`` run.
+            model: If the checkpoint contains a model state dict, and not
+                the model itself, then the state dict will be loaded to this
+                ``model``.
+        """
+        checkpoint_dict = checkpoint.to_dict()
+        preprocessor = checkpoint_dict.get(PREPROCESSOR_KEY, None)
+        if MODEL_KEY not in checkpoint_dict:
```

Comment:
Can we move some this code (like the key validation) into superclasses, like DataParallelTrainer?

Reply:
Done, PTAL!